### PR TITLE
Sync likes.php module file from WP.com, minor fixes to CSS

### PR DESCRIPTION
**Pull request by lancewillett**
A few minor changes, syncing recent fixes from WP.com version.

Note: don't merge until after 4.5 is ready in core, since this removes compat code.

The CSS change is minor, but fixes invalid selector.

### CODE REVIEW

File: modules/likes.php [hunk 1]
```diff
@@ -1020,18 +994,10 @@ function get_options() {
 	 * similar logic and filters apply here, too.
 	 */
 	function is_likes_visible() {
-
 		global $post, $wp_current_filter;              // Used to apply 'sharing_show' filter
-		// @todo: Remove this block when 4.5 is the minimum
```
Comment (kraftbj) [inline modules/likes.php]: We can't merge this yet. 4.5 won't be the minimum version for Jetpack until the release after 4.6 ships.

Comment (lancewillett) [inline modules/likes.php]: OK, I can back that out for this pull request. Thanks!

**Comment by lancewillett** [comment]
@kraftbj How does one stop the madness with the Test on WP.com label? :)

**Comment by lancewillett** [comment]
I updated the commit, the pre-4.5 compat is removed from the changes.

**Comment by lancewillett** [comment]
omg. I give up on the auto-labeling.

**Comment by kraftbj** [comment]
@lancewillett Don't remove it. :) I don't know technically how it was setup, but it forces itself on any PR touched a dotcom-sync'd file. The additional `Tested on WP.com` label confirms that it has been tested.

**Comment by lancewillett** [comment]
Cool. :)

**Comment by samhotchkiss** [comment]
@lancewillett -- there's no auto-labeling, I'm just always watching ;-)

**Comment by kraftbj** [comment]
Rebased so it'll merge cleanly.

**Comment by jeherve** [comment]
This works well in my tests. 🚢

**Pull request merged**